odoo_install.sh

```diff
@@ -38,7 +38,10 @@ OE_CONFIG="${OE_USER}-server"
 WEBSITE_NAME="_"
 # Set the default Odoo longpolling port (you still have to use -c /etc/odoo-server.conf for example to use this.)
 LONGPOLLING_PORT="8072"
-
+# Set to "True" to install certbot and have ssl enabled, "False" to use http
+ENABLE_SSL="True"
```

Comment:
Let's set this to `False` by default and maybe add a link to the certbot docs

Reply:
@Yenthe666 Unless nginx is true this being true wont affect any changes. It's always better to have https than not have it right. Also I can add another check on the website name not = '_'

Reply:
okay sounds good :)